src/pages/AuthPage/ui/auth-ui-form.tsx

```diff
@@ -61,7 +64,7 @@ export function AuthUiForm(props: PaperProps) {
 
           <TextInput
             required
-            label="Email"
+            label={t('auth.login')}
             placeholder="hello@mantine.dev"
             value={form.values.email}
             onChange={(event) => form.setFieldValue('email', event.currentTarget.value)}
```

Comment:
Why don't you use [getinputprops ](https://mantine.dev/form/get-input-props/#getinputprops-handler) here and all forms?